### ISSUE TITLE
Cypress retries on specific target

### ISSUE THREAD

**Issue by Livian95** [cypress-io/cypress#6931]
Current behavior:

I want a test to be retried if another fails, but with the steps left from the one that failed, so, to be more specific :

```js
describe('Something', () => {
  Cypress.env('RETRIES', 3)

  it('x', () => {
    login()
   // Do something here that is necessary for the next test
  })

  it('y', () => {
    loginAsAnotherUser()
    //Test fails because of flakyness in CI, on different steps here, the retry will then come, after a timeout, but it will only retry this second test called 'y'
  })
})
```

### Desired behavior:

```js
describe('Something', () => {
  Cypress.env('RETRIES', 3)

  it('x', () => {
    login()
   // Do something here that is necceserary for the next test
  })

  it('y', () => {
    loginAsAnotherUser()
    //Test steps execute here, because the fleakyness of CI while running in parallel sometimes this test fails, random reassons, for different method calls. What I want is if anything from this 'y' test will fail, I want the test called 'x' test to execute again and then 'y' again. In this 'y' IT, so i won t have failing tests on my dashboard and I will manage a green pipeline if there are not any bugs,
  })
})
```

In short, because how my app is built, i always need to log in with different users at the beginning of each test (it). But the second test is dependent on the first one, if the second test fails, the retry will intervene , but with no actual impact, because retrying is redundant if prior test is not executed again. So if the second test fails, I want to point for the first test to retry ( I want to extrapolate this thinking to test suits that have over 10 tests, but for example purpose only I want an idea for this case with only 2 tests ). The main idea behind is that I will have more stability for my CI/CD.
Thank you very much guys

Cypress 4.2.0, MacOS->Catalina, Chrome

**Comment by jennifer-shehane** [comment]
Test retries is not currently supported in Cypress (you're using [cypress-plugin-retries](https://github.com/Bkucera/cypress-plugin-retries) to achieve this), but we are working on it as part of #1313 - I suggest you voice your wants for this implementation in that issue. 

Duplicate of #1313
